### ISSUE TITLE
Can't load with gulp-load-plugins

### ISSUE THREAD

**Issue by movie4** [dustinspecker/gulp-modify-css-urls#3]
Thanks for amazing plugin!

```
var gulp = require('gulp'),
      plugins = require('gulp-load-plugins')({ pattern: ['gulp-*', 'gulp.*'] });
...
.pipe(plugins.modifyCssUrls({ }))
```

```
[17:35:20] Error: Cannot find module 'gulp-modify-css-urls' from '/Volumes/Work/ITCase/itcase-logos'
    at Function.module.exports [as sync] (/Volumes/Work/ITCase/itcase-logos/node_modules/resolve/lib/sync.js:33:11)
    at requireFn (/Volumes/Work/ITCase/itcase-logos/node_modules/gulp-load-plugins/index.js:42:25)
    at Object.defineProperty.get (/Volumes/Work/ITCase/itcase-logos/node_modules/gulp-load-plugins/index.js:65:18)
    at Gulp.<anonymous> (/Volumes/Work/ITCase/itcase-logos/gulp-tasks/css.js:72:20)
    at module.exports (/Volumes/Work/ITCase/itcase-logos/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Volumes/Work/ITCase/itcase-logos/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Volumes/Work/ITCase/itcase-logos/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Volumes/Work/ITCase/itcase-logos/node_modules/orchestrator/index.js:134:8)
    at /Users/Arkasha/.nvm/versions/node/v4.0.0/lib/node_modules/gulp-cli/index.js:133:20
    at doNTCallback0 (node.js:407:9)

```

Use npm 3.3.10
How fix it?

Thanks

**Comment by movie4** [comment]
https://github.com/jackfranklin/gulp-load-plugins/issues/89#issuecomment-153731244

**Comment by stoikerty** [comment]
This issue just made us unable to deploy onto any boxes, although the dependecy for `gulp-modify-css-urls` didn't change.

The issue was urgent for us so I decided to switch to [`gulp-replace`](https://www.npmjs.com/package/gulp-replace) for our CSS url-changing needs.

**Comment by dustinspecker** [comment]
Fixed in `v0.2.2`. Very sorry about that guys.

**Comment by movie4** [comment]
Thanks for quick fix!
